packages/agoric-cli/src/start.js

```diff
@@ -60,9 +61,16 @@ export default async function startMain(progname, rawArgs, powers, opts) {
     nodeDebugEnv.SWINGSET_WORKER_TYPE = 'local';
   }
 
+  const agServersPrefix = opts.sdk
+    ? undefined
+    : path.resolve(`_agstate/agoric-servers/node_modules/@agoric`);
+  const getBinaries = opts.sdk
+    ? getSDKBinaries
+    : () => getSDKBinaries({ goPfx: agServersPrefix, jsPfx: agServersPrefix });
+
```

Comment:
I assume we're not concerned about multiple versions of these packages getting installed? Or getting installed in another folder than the project root's `node_modules`?

Reply:
Correct.  The binaries must be installed only within the package directory.  Mutating global state is a no-no.

Reply:
I mean that in dapps, `@agoric/cosmic-swingset` is a dependency of the `_agstate/agoric-servers` package, so technically a package manager is free to put it and its dependencies (`@agoric/cosmos`) under that project's `node_modules` only instead of the root level. However I suppose if that happens, it'll just break.

Reply:
> I mean that in dapps, `@agoric/cosmic-swingset` is a dependency of the `_agstate/agoric-servers` package, so technically a package manager is free to put it and its dependencies (`@agoric/cosmos`) under that project's `node_modules` only instead of the root level. However I suppose if that happens, it'll just break.

Ah, I see.  The first time they do an `agoric install <DIST-TAG>`, it will add the `@agoric/cosmic-swingset` and `@agoric/solo` dependencies to the `./package.json`, so this shouldn't be an issue (for long).